[2025-03-11T14:34:35.623+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.insert_employees manual__2025-03-11T14:34:32.270203+00:00 [queued]>
[2025-03-11T14:34:35.632+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: project2.insert_employees manual__2025-03-11T14:34:32.270203+00:00 [queued]>
[2025-03-11T14:34:35.633+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-03-11T14:34:35.633+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-03-11T14:34:35.634+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-03-11T14:34:35.645+0000] {taskinstance.py:1300} INFO - Executing <Task(PostgresOperator): insert_employees> on 2025-03-11 14:34:32.270203+00:00
[2025-03-11T14:34:35.650+0000] {standard_task_runner.py:55} INFO - Started process 3955 to run task
[2025-03-11T14:34:35.652+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'project2', 'insert_employees', 'manual__2025-03-11T14:34:32.270203+00:00', '--job-id', '239', '--raw', '--subdir', 'DAGS_FOLDER/project2.py', '--cfg-path', '/tmp/tmp38877cj9']
[2025-03-11T14:34:35.653+0000] {standard_task_runner.py:83} INFO - Job 239: Subtask insert_employees
[2025-03-11T14:34:35.707+0000] {task_command.py:388} INFO - Running <TaskInstance: project2.insert_employees manual__2025-03-11T14:34:32.270203+00:00 [running]> on host 2ec744569db4
[2025-03-11T14:34:35.776+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project2
AIRFLOW_CTX_TASK_ID=insert_employees
AIRFLOW_CTX_EXECUTION_DATE=2025-03-11T14:34:32.270203+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-03-11T14:34:32.270203+00:00
[2025-03-11T14:34:35.777+0000] {sql.py:254} INFO - Executing: CALL update_employees();
[2025-03-11T14:34:35.785+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-11T14:34:35.969+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2025-03-11T14:34:35.986+0000] {sql.py:375} INFO - Running statement: CALL update_employees();, parameters: None
[2025-03-11T14:34:36.004+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=project2, task_id=insert_employees, execution_date=20250311T143432, start_date=20250311T143435, end_date=20250311T143436
[2025-03-11T14:34:36.025+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-03-11T14:34:36.051+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
